notebooks/1_spikesorting.ipynb

```diff
@@ -4,26 +4,16 @@
    "cell_type": "markdown",
```

Comment:
<p>why did we just define those params if it is already in the table?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/LorenFrankLab/nwb_datajoint/pull/109/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='57'/>

Reply:
<p>I was trying to include an example of how they would do it themselves, so they can change for their own data. I can make a dummy entry and have it clear after every run, so they can insert it for practice. What are your thoughts?</p><p> </p>

Reply:
<p>I think by this point the user should be familiar with basic operations in datajoint like inserting entries to a manual table, so maybe we can just skip this / relegate it to the first tutorial.</p>